src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -1298,3 +1298,34 @@ def get_user_list(self, zoneName=None):
         multisite_instance = RgwMultisite()
         result = multisite_instance.get_user_list(zoneName)
         return result
+
+
+@APIRouter('/rgw/topic', Scope.RGW)
+class RgwTopic(RESTController):
+
+    def create(self, daemon_name=None, owner=None,
```

Comment:
Can you add documentation to all the other endpoints (list, get and delete) as I put below.
Also rearrange the params in a way that daemon_name and owner will be the last probably but `topic_name` should be the first and it should be a required parameter instead of an optional param.

Btw, we can call `topic_name` just `name`. Its pretty much given this endpoint is for topic management so we don't need to add the `topic_` prefix anywhere.

```suggestion
    @EndpointDoc(
            "Create a new RGW Topic",
            parameters={
                 "name": Param(str, "Name of the topic")
                 .....
                 ....
            }
    )
    def create(self, daemon_name=None, owner=None,
```

Reply:
changes are done